src/reactionsystem_conversions.jl

```diff
@@ -450,6 +450,17 @@ diff_2_zero(expr) = (Symbolics.is_derivative(expr) ? 0 : expr)
 
 COMPLETENESS_ERROR = "A ReactionSystem must be complete before it can be converted to other system types. A ReactionSystem can be marked as complete using the `complete` function."
 
+# Used to, when required, display a warning about conservation law removeal and remake.
+function check_cons_warning(remove_conserved, remove_conserved_warn)
+    (remove_conserved && remove_conserved_warn) || return
+    @warn "You are creating a system while eliminating conserved quantities. While this is possible, 
+        if you use the created system to create a problem (e.g. an `ODEProblem`), you *should not*
+        modify that problem's species values (e.g. using `remake`). Modification of parameter values 
+        is still possible. You might get this warning when creating a problem directly.
```

Comment:
```suggestion
        is still possible, and directly setting numerical values for any conservation law constants will work. You might get this warning when creating a problem directly.
```
We should check this statement is correct.